client/src/components/StyleSelector.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { FaCheckCircle } from 'react-icons/fa';
+import StyleBubble from './StyleBubble';
+
+
+
+
+function StyleSelector(props) {
+
+ 
+  const { styles, selectedStyle } = props;
+
+  if (styles.length > 0) {
+    const wrappedStyles = styles.map((style) => (
+      <div>
+        {style.style_id === selectedStyle.style_id && <FaCheckCircle />}
```

Comment:
while cool, this syntax hurts the readability of your code. use an if statement or the ternary operator instead